apps/mosaico/src/Article/Article.purs

```diff
@@ -77,7 +78,7 @@ articleComponent = do
           }
     state /\ setState <- useState initialState
 
-    useEffectOnce do
+    useEffect props.uuid do
```

Comment:
I suppose this fixed rendering of clicked articles? It's weird, as `props` should not ever change 😄 I mean, if the `props` change, that means the component is re-mounted with another set of `props`.

Reply:
Or is this some react shenanigans I'm not aware of?

Reply:
This is the behaviour we observed: `useEffectOnce`'s callback was effectively being called only once, while `useEffect`'s callback was called when the watched properties of the component changed.

If we look at [`useEffectOnce`'s description from **pursuit**](https://pursuit.purescript.org/packages/purescript-react-basic-hooks/7.0.0/docs/React.Basic.Hooks#v:useEffectOnce) it says:

> Like `useEffect`, but the effect is only performed a single time per component instance.

Indeed the component instance is always the same, because what we do when initializing  the `Mosaico` component is effectfully obtaining the `Article` component and saving it in the state of the `Mosaico` component. Then, when the user wants to read a different article, we pass a different set of properties to that same component instance.

Thus `useEffectOnce` cannot work in our case, but `useEffect` does.